core/shared/src/main/scala-3/zio/prelude/Liftables.scala

```diff
@@ -130,7 +130,7 @@ trait Liftables {
       case _: Short  => summon[Numeric[Short]].asInstanceOf[Numeric[Any]]
       case _: Byte   => summon[Numeric[Byte]].asInstanceOf[Numeric[Any]]
       case _: Char   => summon[Numeric[Char]].asInstanceOf[Numeric[Any]]
-      case other     => report.throwError(s"NO NUMERIC FOR $other")
+      case other     => report.errorAndAbort(s"NO NUMERIC FOR $other")
```

Comment:
Was generating the following scalac warning:
```scala
[warn] -- Deprecation Warning: /Users/jules/conduktor/workspace/zio-prelude/core/shared/src/main/scala-3/zio/prelude/Liftables.scala:133:31 
[warn] 133 |      case other     => report.throwError(s"NO NUMERIC FOR $other")
[warn]     |                        ^^^^^^^^^^^^^^^^^
[warn]     |method throwError in trait reportModule is deprecated since 3.1.0: Use errorAndAbort
```